src/test/java/com/agnieszkapawska/flashcards/api/controllers/FlashcardControllerTest.java

```diff
@@ -1,39 +1,54 @@
 package com.agnieszkapawska.flashcards.api.controllers;
 
+import com.agnieszkapawska.flashcards.FlashcardAndQuestionTagAbstractTests;
 import com.agnieszkapawska.flashcards.FlashcardsApplicationAbstractTests;
 import com.agnieszkapawska.flashcards.domain.dtos.FlashcardSaveDto;
 import com.agnieszkapawska.flashcards.domain.dtos.FlashcardSaveResponseDto;
 import com.agnieszkapawska.flashcards.domain.exceptions.EntityNotCreatedException;
+import com.agnieszkapawska.flashcards.domain.facades.FlashcardFacade;
 import org.junit.Assert;
+import org.junit.Before;
 import org.junit.Test;
+import org.springframework.boot.test.mock.mockito.MockBean;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.Mockito.doThrow;
 import static org.mockito.Mockito.when;
 
-public class FlashcardControllerTest extends FlashcardsApplicationAbstractTests {
+public class FlashcardControllerTest extends FlashcardAndQuestionTagAbstractTests {
+    private String url;
+    @MockBean
+    private FlashcardFacade flashcardFacade;
+
+    @Before
+    public void setUp() {
+        super.setUp();
+        url = FlashcardsApplicationAbstractTests.baseUrl + "/flashcards";
+    }
 
     @Test
     public void shouldReturnStatusOk_WhenSavingFlashcard() {
+        System.out.println(url);
         //given
-        when(super.flashcardFacade.saveFlashcard(any(FlashcardSaveDto.class))).thenReturn(super.flashcardSaveResponseDto);
+        when(flashcardFacade.saveFlashcard(any(FlashcardSaveDto.class))).thenReturn(super.flashcardSaveResponseDto);
```

Comment:
Maybe let's move `.thenReturn...` . to next line. It should be more readable then.